algorithms/yiddish/stemmer.tt

```diff
@@ -138,8 +130,8 @@ be generated with this algorithm.
     <li>Stressed: אױס, אױפ, אומ, אונטער, איבער, אײֲנ, אָנ, אָפּ, בײַ, פאָר, צו.</li>
```

Comment:
There seem to be 24 "Free stressed" prefixes here, but in the code and the comment in the code there seem to be 25.

(I've tried to compare to find which is the extra one, but my attempts to convert don't seem to give lists which even slightly match - probably I'm not removing the "niked added for clarity" suitably.)